server/CGameHandler.cpp

```diff
@@ -4858,6 +4859,25 @@ bool CGameHandler::handleDamageFromObstacle(const CStack * curStack, bool stackI
 
 					if(oneTimeObstacle)
 						removeObstacle(*obstacle);
```

Comment:
Put `{}` around because else clause in not one line.

Reply:
Updated.